src/App.jsx

```diff
@@ -17,7 +17,7 @@ import { PDFViewer, StyleSheet } from '@react-pdf/renderer';
 import './App.css';
 import DonationForm from './components/DonationForm/DonationForm';
 import DropZone from './components/DropZone/DropZone';
-import uploadImage from './utils/FurnitureUtils'; // was originally 'FurnitureUtils' should probably be this
+import uploadImage from './utils/FurnitureUtils';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './utils/FurnitureUtils'.